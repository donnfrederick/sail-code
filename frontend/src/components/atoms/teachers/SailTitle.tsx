import * as React from 'react'
import styled from 'styled-components'
import resolvePath from 'utils/resolvePath'

export default () => {
  return (
    <Container>
      <Title>
        <Logo src={resolvePath.image('common/sail@3x.png')} />
      </Title>
      <SubTitle>さあ、世界へ船出しよう</SubTitle>
    </Container>
  )
}

const Container = styled.div`
  margin-bottom: 0;
`

const Title = styled.h1`
  height: 180px;
  margin: 0 0 36px;
  text-align: center;
`

const Logo = styled.img`
  height: 100%;
`

const SubTitle = styled.h2`
  margin: 0;
  font-size: 38px;
  font-weight: bold;
  font-style: normal;
  font-stretch: normal;
  letter-spacing: 0px;
  text-align: center;
  color: #000000;
`
