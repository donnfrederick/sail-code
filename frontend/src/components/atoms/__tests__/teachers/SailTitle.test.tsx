import SailTitle from 'components/atoms/teachers/SailTitle'
import * as React from 'react'
import * as renderer from 'react-test-renderer'

test('render <SailTitle />', () => {
  const tree = renderer.create(<SailTitle />).toJSON()
  expect(tree).toMatchSnapshot()
})
