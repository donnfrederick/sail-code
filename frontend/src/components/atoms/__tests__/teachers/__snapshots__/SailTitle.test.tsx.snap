// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render <SailTitle /> 1`] = `
<div
  className="sc-bdVaJa ezlkVu"
>
  <h1
    className="sc-bwzfXH gjfzmg"
  >
    <img
      className="sc-htpNat kQspvn"
      src="/assets/img/common/sail@3x.png"
    />
  </h1>
  <h2
    className="sc-bxivhb liuDuV"
  >
    さあ、世界へ船出しよう
  </h2>
</div>
`;
